Parse/cloud/main.js

```diff
@@ -355,6 +364,59 @@ Parse.Cloud.define("postVent", async (request) => {
   return;
 });
 
+Parse.Cloud.define("createGroup", async (request) => {
+  const Group = Parse.Object.extend("GroupDetails");
+  const group = new Group();
+  const authorId = request.params.authorId;
+  var groupMembershipIds = request.params.groupMembershipIds;
+  var groupName = request.params.groupName;
+
+  const userQuery = new Parse.Query("User");
+  userQuery.equalTo("objectId", authorId);
+  const user = await userQuery.first();
+
+  if (user == undefined) {
+    return;
```

Comment:
Rather than doing a return, which is like a silent error that can cause very confusing debugging, can you throw an error here as well?